installing/installing_aws/installing-aws-customizations.adoc

```diff
@@ -30,7 +30,7 @@ in the AWS documentation. You can supply the keys when you run the installation
 program.
 ====
 * If you use a firewall, you must
-xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to access Red Hat Insights].
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow access to the sites] required for your {product-title} cluster.
```

Comment:
s/required for your {product-title} cluster./that your cluster requires access to.